exercises/spiral-matrix/description.md

```diff
@@ -0,0 +1,22 @@
+Given a number, return a matrix of numbers in spiral order.
+
+The matrix should be filled with natural numbers, starting from 1
+in the top-left corner, increasing in a inward, clockwise spiral order,
+like these examples:
+
+- Spiral matrix of size 2
+
+    ```plain
+    1 2 3
+    8 9 4
+    7 6 5
+    ```
+
+- Spiral matrix of size 5
```

Comment:
`Spiral matrix of size 5` - where does the 5 come from here? The given matrix is 4x4!